### ISSUE TITLE
Baseline training results (Smooth loss increasing)

### ISSUE THREAD

**Issue by leyuan-sun** [JiawangBian/SC-SfMLearner-Release#78]
I did not change any hyperparameters, however, the smoothness loss is increasing when training your provided data as the images shown, is it correct?
![Screenshot from 2021-05-07 17-38-31](https://user-images.githubusercontent.com/41459267/117423277-83059d00-af5b-11eb-82bc-d80f5a4abb1e.png)
![Screenshot from 2021-05-07 17-38-58](https://user-images.githubusercontent.com/41459267/117423280-839e3380-af5b-11eb-9683-2074d54d313c.png)

**Comment by JiawangBian** [comment]
yes，it is correct. it means more and more sharp

**Comment by leyuan-sun** [comment]
> yes，it is correct. it means more and more sharp

I can somehow understand that it means sharper, but Shouldn't the loss finally converge to 0 in common training?

**Comment by JiawangBian** [comment]
it is not

**Comment by leyuan-sun** [comment]
Thank u for your reply, but I still can not understand...

**Comment by leyuan-sun** [comment]
> it is not

One more question, should other losses including total loss finally be converged to 0?

**Comment by JiawangBian** [comment]
shouldn't

**Comment by leyuan-sun** [comment]
> shouldn't

what is the expected loss finally individually?

**Comment by JiawangBian** [comment]
it would converge to some value，but not 0

**Comment by leyuan-sun** [comment]
Thank u so much, I will plot all final loss curves as a reference baseline for others and then closed this issue.

**Comment by leyuan-sun** [comment]
For everyone as the baseline to compare, this is the training result as the same hyperparameters in train_resnet50_pose256 script.
@JiawangBian If you think something is incorrect(loss curves and predicted depth), plz let me know.
![Screenshot from 2021-05-11 09-24-52](https://user-images.githubusercontent.com/41459267/117740572-af5f3900-b23b-11eb-9962-f4b1bc3b7c96.png)
![Screenshot from 2021-05-11 09-25-15](https://user-images.githubusercontent.com/41459267/117740576-b128fc80-b23b-11eb-870c-b54c79af72c5.png)
![Screenshot from 2021-05-11 09-25-27](https://user-images.githubusercontent.com/41459267/117740578-b128fc80-b23b-11eb-9f53-ae0103bb498a.png)
![Screenshot from 2021-05-11 09-25-44](https://user-images.githubusercontent.com/41459267/117740579-b1c19300-b23b-11eb-8a9a-c58bcb035545.png)
![Screenshot from 2021-05-11 09-26-03](https://user-images.githubusercontent.com/41459267/117740581-b1c19300-b23b-11eb-9241-8a768908734e.png)
![Screenshot from 2021-05-11 09-26-16](https://user-images.githubusercontent.com/41459267/117740583-b1c19300-b23b-11eb-9343-fe8bfa2e33c8.png)
![Screenshot from 2021-05-11 09-26-33](https://user-images.githubusercontent.com/41459267/117740584-b25a2980-b23b-11eb-87f1-f8863d766cc5.png)
![Screenshot from 2021-05-11 09-26-44](https://user-images.githubusercontent.com/41459267/117740585-b25a2980-b23b-11eb-8830-ae7de3ee518c.png)
![Screenshot from 2021-05-11 09-27-09](https://user-images.githubusercontent.com/41459267/117740586-b2f2c000-b23b-11eb-86c5-0edea4b210aa.png)

**Comment by JiawangBian** [comment]
it looks correct.
